huggingface/tensorflow/inference/docker/2.4/py3/cu110/Dockerfile.gpu

```diff
@@ -1,5 +1,125 @@
-# https://github.com/aws/deep-learning-containers/blob/master/available_images.md
-# refer to the above page to pull latest Tensorflow image
+FROM nvidia/cuda:11.0-cudnn8-runtime-ubuntu18.04
 
-# docker image region us-west-2
-FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/tensorflow-inference:2.4.1-gpu-py37-cu110-ubuntu18.04
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG MMS_VERSION=1.1.4
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.7.10
+ARG OPEN_MPI_VERSION=4.0.1
+# HF ARGS
+ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.4_aws/20210127-150238/gpu/py37/cu110/tensorflow_gpu-2.4.1-cp37-cp37m-manylinux2010_x86_64.whl
+ARG TRANSFORMERS_VERSION
+ARG HF_INFERENCE_TOOLKIT_VERSION=0.0.1.dev0
+
+
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1 \
+    LD_LIBRARY_PATH="/opt/conda/lib/:${LD_LIBRARY_PATH}:/usr/local/lib" \
+    PYTHONIOENCODING=UTF-8 \
+    LANG=C.UTF-8 \
+    LC_ALL=C.UTF-8 \
+    TEMP=/home/model-server/tmp \
+    DEBIAN_FRONTEND=noninteractive
+
+ENV PATH /opt/conda/bin:$PATH
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+    ca-certificates \
+    build-essential \
+    openssl \
```

Comment:
Need to ensure openssl in GPU images don't trigger alarms in security scan.

Reply:
The `apt-get update && apt-get install openssl` as written here should update the openssl version to the latest in every build.